packages/react/src/components/Button/BaseButton.tsx

```diff
@@ -66,6 +67,10 @@ export class BaseButton extends React.Component<IBaseButtonProps, IBaseButtonSta
     split: false,
   };
 
+  // needed to access registeredProviders when manually setting focus visibility
+  public static contextType = FocusRectsContext;
+  public context: React.ContextType<typeof FocusRectsContext>;
```

Comment:
You could explicitly import `IFocusRectsContext` and use it here so that it isn't imported from 2 places in the .api file.

Reply:
ah, good call!